src/tfvc/scm/model.ts

```diff
@@ -175,4 +179,20 @@ export class Model {
         }
         return result;
     }
+
+    //When files are deleted in the VS Code Explorer, they are marked as Deleted but as candidate changes
+    //So we are looking for those and then running the Delete command on each.
+    private async processDeletes(changes: IPendingChange[]): Promise<void> {
+        let deleteCandidatePaths: string[] = [];
+        for (let index: number = 0; index < changes.length; index++) {
+            let change: IPendingChange = changes[index];
+            if (change.isCandidate && GetStatuses(change.changeType).find((e) => e === Status.DELETE)) {
```

Comment:
You can use change.HasStatus(Status.DELETE) here. I think it does the same thing.